modules/nodes-pods-vertical-autoscaler-using-about.adoc

```diff
@@ -190,7 +192,11 @@ spec:
 
 [NOTE]
 ====
-There must be operating pods in the project before a VPA can determine recommended resources and apply the recommendations to new pods.
+For the VPA to provide you with an accurate recommendation, the VPA must observe pod metrics while the running workload uses resources that are not considered atypical. 
```

Comment:
For me, "not considered atypical" reads awkwardly and emphasizes what not to do. I think it would be better to tell what to do instead of what not to do. Especially since as written, it seems to imply that bad things will happen to your recommendation if you have atypical usage. What we want to convey is that when the VPA observes your workload using resources it typically uses, it will make a higher-quality recommendation.

Reply:
Thanks, @joelsmith . Good point. What about the following slight variation:

> For the VPA to provide you with a more accurate recommendation, the VPA must observe pod metrics on running workload that uses typical resources.